macros/LoadRelatedEvents.ejs

```diff
@@ -1,16 +1,5 @@
 <%
-var el = env.locale;
-var path = "";
-
-switch (el) {
-    case "ja":
-    case "zh-TW":
-        path = "/" + el + "/docs/";
-        break;
-    default:
-        path = "/en-US/docs/"
-}
-
+var path = "/" + env.locale + "/docs/";
```

Comment:
This changes is expected. Any unexpected error should be added by a specific one, otherwise use this default.